game/templates/game/stats.html

```
@@ -65,5 +81,36 @@ <h2>Recent Games</h2>
+    <div style=" background:#16162a; border:1px solid #252545; border-radius:12px; padding:25px; width:500px; max-width:90%; color:white;">
+        <div class="promo-title">
+            Game Summary
+        </div>
+        <div class="summary-card">
+            <p><strong>Mode:</strong> <span id="summaryMode"></span></p>
+            <p><strong>Winner:</strong> <span id="summaryWinner"></span></p>
+            <p><strong>Result:</strong> <span id="summaryReason"></span></p>
+            <p><strong>Move Count:</strong> <span id="summaryMoves"></span></p>
+            <p><strong>Played At:</strong> <span id="summaryDate"></span></p>
+        </div>
+        <button onclick="closeSummary()" class="btn btn-gold">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)